traffic_ops/traffic_ops_golang/riak.go

```diff
@@ -0,0 +1,267 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/lib/go-tc"
+	"github.com/basho/riak-go-client"
+	"github.com/jmoiron/sqlx"
+	"github.com/lestrrat/go-jwx/jwk"
+	"io/ioutil"
+	"net/http"
+)
+
+const RiakPort = 8087
+const cdn_uri_keys_bucket = "cdn_uri_sig_keys" // riak namespace for cdn uri signing keys.
```

Comment:
variables/constants are "camelCase" in Go instead of snake case

Reply:
ok, i'll make these changes